README.md

```diff
@@ -213,6 +213,26 @@ resources in the `"included"` member when the resource names are included in the
   render @posts, include: 'author,comments,comments.author'
 ```
 
+In addition, two types of wildcards may be used. `*` includes one level of associations, and `**` includes all recursively. These can be combined with other paths.
+
+```ruby
+  render @posts, include: '**' # or '*' for a single layer
+```
+
+The following would render posts and include every resource referenced by the posts' authors' comments (recursively). It could be combined, like above, with other paths in any combination desired.
```

Comment:
LGTM other than this. Sorry for being a bit a bit tedious on this PR but I can foresee many issues caused by people misunderstanding this functionality.